unstructured/documents/elements.py

```diff
@@ -614,6 +615,10 @@ class ElementType:
     LIST_ITEM_OTHER = "List-item"
     CHECKED = "Checked"
     UNCHECKED = "Unchecked"
+    CHECK_BOX_CHECKED = "CheckBoxChecked"
+    CHECK_BOX_UNCHECKED = "CheckBoxUnchecked"
+    RADIO_BUTTON_CHECKED = "RadioButtonChecked"
+    RADIO_BUTTON_UNCHECKED = "RadioButtonUnchecked"
```

Comment:
we still need to add mapping of those to `CHECKED/UNCHECKED` below, right?

Reply:
CheckBox subclasses from `Element`, not `Text` - so these have a dedicated "if branch" here:
https://github.com/Unstructured-IO/unstructured/pull/2700/files#diff-9eb4b1248793302fa3b6bc9db25756503ea28f53ca556417c27f0be85d7a709bR164

We have a `checked` attribute in `CheckBox` even if its base class is moved to `Text`, we cannot cover it with standard `Text` elements mapping.